### PR TITLE
[IMP] point_of_sale, *: revamp UI fixes

### PR DESCRIPTION
**Pull request by xlu-odoo**
*: pos_restaurant

This commit introduces a few improvements and UI fixes following the POS redesign.

task-4111569

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/point_of_sale/static/src/app/screens/payment_screen/payment_lines/payment_lines.xml [hunk 1]
```diff
@@ -19,14 +19,10 @@
                                 </div>
                             </div>
                             <t t-if="!line.payment_status or !['done', 'reversed', 'waitingCard', 'waitingCapture'].includes(line.payment_status)">
-                                <button t-if="ui.isSmall" class="btn btn-link mx-2 px-3"
-                                aria-label="Edit" title="Edit" t-on-click="() => this.selectLine(line)">
-                                    <i class="fa fa-2x fa-edit" />
-                                </button>
```
Comment (caburj) [inline addons/point_of_sale/static/src/app/screens/payment_screen/payment_lines/payment_lines.xml:25]: why is this removed?

Comment (xlu-odoo) [reply addons/point_of_sale/static/src/app/screens/payment_screen/payment_lines/payment_lines.xml:25]: FP requested to remove this icon/btn-link. You can still edit by clicking on the line afair.

**Comment by vlst-odoo** [comment]
robodoo r+

**Comment by caburj** [comment]
robodoo r+

**Comment by caburj** [comment]
forced push because there was a conflict
